filebeat.yml

```
@@ -0,0 +1,17 @@
+
+setup.template.settings:
+  index.number_of_shards: 1
+setup.kibana:
+
+processors:
+  - add_host_metadata:
+      when.not.contains.tags: forwarded
+  - add_cloud_metadata: ~
+
+output.logstash:
+  hosts: ['graylog:5044']
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)